core/templates/pages/exploration-editor-page/settings-tab/settings-tab.component.ts

```diff
@@ -408,6 +415,19 @@ angular.module('oppia').component('settingsTab', {
             }
           )
         );
+        ctrl.directiveSubscriptions.add(
+          UserExplorationPermissionsService.onUserExplorationPermissionsFetched
```

Comment:
I was not able to find the place where we are calling emit on this event?

Reply:
Here:
https://github.com/oppia/oppia/blob/a1830603befc330156d40e8c3e79f3f57179f89c/core/templates/pages/exploration-editor-page/services/user-exploration-permissions.service.ts#L53